### DISCUSSION TITLE
Automation of the release process

### DISCUSSION THREAD

**Discussion by slawekjaranowski** [support-and-care/maven-support-and-care#27] [Ideas for Funding]
We have many manual steps, I hope some of them can be automated

https://maven.apache.org/developers/release/maven-project-release-procedure.html

**Comment by khmarbaise** [comment]
Yes good idea.
* Automate the release/staging process
  * for the artifacts (`release:prepare` and `release:perform`)
  * Artifact Staging
  * Site generation and staging of the site
  * Signing might be question about the GPG Key
* Create a web sited (service) which handles the VOTING Process (login for PMC's etc.) and without a login for others users? (Might be checked with ASF if such things is compliant with the ASF rules.. ?

**Comment by olamy** [comment]
some tooling have been recently added in Apache infra https://artifacts.apache.org/
https://lists.apache.org/list?artifacts@infra.apache.org
I haven't used it so I cannot tell but worth to have a look

**Comment by ascheman** [comment]
This discussion has been converted to Epic #63 for tracking.

Closing this discussion.
